### DISCUSSION TITLE
Syntax for preventing Latex formatting?

### DISCUSSION THREAD

**Discussion by hahnbeelee** [remarkjs/remark#1203] [Q&A] [answered]
I have some content where it says "$50 some text $50" and I'm using `remark-math` and everything between the two dollar signs is being formatted to LateX syntax. Is there syntax that prevents this from happening? I looked at the `remark-math` readme but wasn't able to find any explicit syntax to prevent this behavior.

**Comment by ChristianMurphy** [comment, marked as answer]
Hiya @hahnbeelee! 👋 
While not specified in the readme, math-like blocks can be escaped with `\` to prevent them from turning into latex, similar to most other commonmark and gfm constructs which also support `\`.
An example of this would be:
```md
\$50 some text \$50
```

AST generated: https://astexplorer.net/#/gist/86419620b3a99a97a630346e7a8e1958/7d1d9cb4c5dc9de57b2c564c8963c958f7c366a9

**Reply by hahnbeelee** [reply]
Hi Christian thank you for the speedy response! I'll make sure to escape my dollar signs from here on out :D thank you

**Reply by wooorm** [reply]
You can turn singleDollar off btw, then you need two dollars on both sides to get math. And a single dollar will just remain a dollar
